core/src/clients/git.rs

```diff
@@ -507,6 +577,10 @@ impl Git {
         // disable clone protection
         cmd.env("GIT_CLONE_PROTECTION_ACTIVE", "false");
 
+        if opts.lfs_mode == LfsMode::Stubs {
+            cmd.env("GIT_LFS_SKIP_SMUDGE", "1");
```

Comment:
Awesome. I can't help but think this new opt is going to help us with a lot of stuff in the future.